examples/django/README.rst

```diff
@@ -34,3 +47,14 @@ Starting the worker
 .. code-block:: bash
 
     $ celery -A proj worker -l info
+
+Running a task
+===================
+
+.. code-block:: bash
+    
+    $ python ./manage.sh shell
```

Comment:
isn't it manage.py? and if you write ./manage.py shell, no need to put python infront